### ISSUE TITLE
no-unsupported-features/node-builtin should detect unsupported usage of `fs.rmDir`

### ISSUE THREAD

**Issue by Skn0tt** [mysticatea/eslint-plugin-node#221]
The following is supported from Node v12 onwards:

```js
fs.rmDir("myRandomDir", { recursive: true });
```

It'd be awesome if `no-unsupported-features/node-builtin` caught this.

For reference: https://github.com/blitz-js/blitz/pull/454
